torch/_dynamo/eval_frame.py

```diff
@@ -166,6 +184,40 @@ def _callback_from_stance(callback):
             )
 
         return callback
+    elif _stance.stance == "eager_then_compile":
+        if callback not in (False, None):
+
+            def foo(*args, **kwargs):
```

Comment:
Match the convert frame function protocol here for readability

Reply:
Hmmm but I need args kwargs later on in the function. One unpacking seems nicer and much less verbose than unpacking everything?

Reply:
Ok, fair